_data-prepper/managing-data-prepper/source-coordination.md

```diff
@@ -0,0 +1,173 @@
+| `ttl`                              | Duration | Optional. The duration of the time to live for the items in the table. This TTL is extended by this duration when an update is made to the item. Defaults to no TTL being used on the table. |
+  
+Full configuration example
+
+```yaml
+source_coordination:
+  store:
+     dynamodb:
+       table_name: "DataPrepperSourceCoordinationStore"
+       region: "us-east-1"
+       sts_role_arn: "arn:aws:iam::870201406020:role/SourceCoordinationTableRole"
+       ttl: "P7D"
+       skip_table_creation: true
+```
+
+##### In memory store (default)
+
+No additional configurations available. This store is only useful for single-node environments of Data Prepper. Full configuration example:
+
+```yaml
+source_coordination:
+  store:
+    in_memory:
+```
+
+
+## Sources using source coordination
+* S3 source with `scan`, where each partition is an S3 object identified in the format of `bucket|key`
+
+## Metrics
+
+Source coordination has metrics that are interpreted differently depending on which source is configured. The format of all metrics will be
+`<sub-pipeline-name>_source_coordinator_<metric-name>`, and sub-pipeline name is used to identify which source these metrics are for, as each sub-pipeline
+is unique to a single source.
+
+Progress metrics
+
+* `partitionsCreatedCount` - The number of partition items that have been created. For S3 scan, this is the number of objects that have had partitions created for them.
+* `partitionsCompleted` - The number of partitions that have been fully processed and marked as COMPLETED. For S3 scan, this is the number of objects that have been processed.
+* `noPartitionsAcquired` - The number of times that a node has attempted to acquire a partition to do work on, but has found no available partitions in the store. Can be used to indicate that there is no more data coming in the source.
+* `partitionsAcquired` - The number of partitions that have been acquired by nodes to do work on. In non-error scenarios, this should be equal to the number of partitions created.
+* `partitionsClosed` - The number of partitions that have been marked as CLOSED. Only applicable to sources that use the CLOSED functionality
+
+Error metrics
+
+* `partitionNotFoundErrors` - Indicates that a partition item that is actively owned by a node does not have a corresponding store item. This should only happen if the item in the table has been manually deleted
+* `partitionNotOwnedErrors` - Indicates an issue where the node that owns a partition loses ownership due to the partition ownership timeout expiring. Unless the source is able to checkpoint with `saveState`, this will result in duplicate processing
+* `partitionUpdateErrors` - The number of errors that were received when an update to the store for this partition item failed. Is prefixed with either `saveState`, `close`, or `complete` to indicate which update action is failing.
+
+
+## Using source coordination for new sources
```

Comment:
Would this section only be of interest to Data Prepper developers, or would end-users also want to develop a coordinator for a new sources?

Reply:
Developers only

Reply:
I'm going to remove this section then.